### ISSUE TITLE
Unable to see the component on the page

### ISSUE THREAD

**Issue by rishabhchd19** [lingwooc/cancel-magento2#1]
Hi. I am unable to see any option "Cancel" order in users "customer's" page. Magento CE 2.2.5

**Comment by lingwooc** [comment]
This is either because 
1. magento have changed those pages and the module is out of date OR
2. the order isn't in the correct state. 

The answer to this question explains the states.
https://magento.stackexchange.com/questions/135281/magento-2-is-it-possible-to-add-order-cancellation-functionality-in-frontend

If it's been paid for you can't cancel it. If it's definitively not been then maybe my templates/layouts are out of date.

**Comment by rishabhchd19** [comment]
Yes may be due to status "Processing" this may not be showing. Let me check. Thanks.
